receive.c

```diff
@@ -352,7 +384,8 @@ int main(int argc, char **argv) {
 				printf("\t -P --port=xxxx\t\t\tconnect to server port\n");
 				printf("\t -I --instance=name\t\tconnect to pilight instance\n");
 				printf("\t -s --stats\t\t\tshow CPU and RAM statistics\n");
-				printf("\t -F --filter=protocol\t\tfilter out protocol(s)\n");
+				printf("\t -F --filter=protocol\t\tfilter for protocol(s)\n");
+				printf("\t -X --exclude=protocol\t\texclude protocol(s) from output\n");
```

Comment:
The description is not quite uniform. The `-X` says  `... from output`, while the `-F` does not.